api/tests/opentrons/protocol_engine/commands/test_blow_out.py

```diff
@@ -0,0 +1,79 @@
+"""Test blow-out command."""
+from decoy import Decoy
+from dataclasses import field
+from typing import cast
+
+from opentrons.protocol_engine import WellLocation, WellOrigin, WellOffset
+from opentrons.protocol_engine.state import StateView, HardwarePipette
+from opentrons.protocol_engine.commands import (
+    BlowOutResult,
+    BlowOutImplementation,
+    BlowOutParams,
+)
+from opentrons.protocol_engine.execution import (
+    MovementHandler,
+)
+from opentrons.types import Mount
+from opentrons.hardware_control.dev_types import PipetteDict
+from opentrons.hardware_control import HardwareControlAPI
+
+
+async def test_blow_out_implementation(
+    decoy: Decoy,
+    state_view: StateView,
+    hardware_api: HardwareControlAPI,
+    movement: MovementHandler,
+) -> None:
+    """Test BlowOut command execution."""
+    subject = BlowOutImplementation(
+        state_view=state_view, movement=movement, hardware_api=hardware_api
+    )
+
+    left_config: PipetteDict = field(
+        default_factory=lambda: cast(
+            PipetteDict, {"name": "p300_single", "pipette_id": "123"}
+        )
+    )
+    right_config: PipetteDict = field(
+        default_factory=lambda: cast(
+            PipetteDict, {"name": "p300_multi", "pipette_id": "abc"}
+        )
+    )
```

Comment:
`field` should only ever be used as a default parameter value for a dataclass

```suggestion
    left_config = cast(PipetteDict, {"name": "p300_single", "pipette_id": "123"})
    right_config = cast(PipetteDict, {"name": "p300_multi", "pipette_id": "abc"})
```